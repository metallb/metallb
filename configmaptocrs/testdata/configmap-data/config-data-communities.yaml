bgp-communities:
    no-export: 65535:65281
    my-community: 1234:5678
