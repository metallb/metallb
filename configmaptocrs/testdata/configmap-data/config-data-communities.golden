# This was autogenerated by MetalLB's custom resource generator.
apiVersion: metallb.io/v1beta1
kind: Community
metadata:
  creationTimestamp: null
  name: communities
  namespace: metallb-system
spec:
  communities:
  - name: my-community
    value: 1234:5678
  - name: no-export
    value: 65535:65281
status: {}
---
